docs/examples/endpoint/README.md

```diff
@@ -0,0 +1,57 @@
+# Ingress
```

Comment:
Maybe the headline should be something else? like endpoint? or both?

Reply:
changed to endpoint